requirements.txt

```diff
@@ -1,4 +1,7 @@
-linode-api4>=5.13.1
+# TODO: Blocked by SDK release; revert before merging to dev
+# linode-api4>=5.13.1
+git+https://github.com/linode/linode_api4-python@dev
```

Comment:
The unordered annotation is a super recent addition so we'll need to cut a new Python SDK release before merging this PR